src/drt/src/gc/FlexGC_impl.h

```diff
@@ -140,12 +139,17 @@ class FlexGCWorker::Impl
   Logger* logger_;
   RouterConfiguration* router_cfg_;
   FlexDRWorker* drWorker_;
+  Arena<gcNet> netArena_ = {128};
+  Arena<gcPin> pinArena_ = {512};
+  Arena<gcRect> rectArena_ = {512};
+  Arena<gcSegment> segmentArena_ = {512};
+  Arena<gcCorner> cornerArena_ = {512};
```

Comment:
any comment why these exact block sizes?

Reply:
No particular reason. The new version I just pushed I tested empirically. It's around the optimum for the systems I tested on.